lib/granite/form/model/attributes/base.rb

```diff
@@ -58,7 +58,7 @@ def readonly?
           def inspect_attribute
             value = case read
             when Date, Time, DateTime
```

Comment:
Hey @mpiask, I've set up the environment with Ruby 2.6.9 and
```
export BUNDLE_GEMFILE=gemfiles/rails.5.2.gemfile
```

I've added `raise` above the change, and it looks like this code is **not** called by unit tests.
Could you add a unit test, please?

That's not all, as it looks that `to_fs` method is not present in Rails 5.2 (while it's part of [tests matrix](https://github.com/toptal/granite-form/blob/61278eb318442e2395c7b218998d55c4cd71a96f/.github/workflows/ruby.yml#L13-L19))

`to_formatted_s` looks to be a better choice here:

```
Time.now.to_s(:db)
=> "2023-06-28 09:31:01"

Time.now.to_fs(:db)
NoMethodError: undefined method `to_fs' for 2023-06-28 09:31:04 +0200:Time
...
from (pry):2:in `inspect_attribute'

Time.now.to_formatted_s(:db)
=> "2023-06-28 09:31:08"
```